docs/guides/notifications.md

```diff
@@ -144,16 +164,5 @@ spec:
       namespace: gitops-system
 ```
 
-The secret referenced in the provider is expected to contain a [personal access token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)
-to authenticate with the GitHub API.
-```yaml
-apiVersion: v1
-kind: Secret
-metadata:
-  name: github
-  namespace: gitops-system
-data:
-  token: <token>
-```
-
+### Health checks
```

Comment:
Need to add some examples how notifications behave when a Kustomization goes from healthy to unhealthy.